### PR TITLE
Auth and feedback

### PR DESCRIPTION
**Pull request by tkellen**
This adds support for adding beta users via an array of emails. 
* to edit users: `npm run edit-secrets`
* to provision users: `npm run provision:[production|staging]`

...also a backend endpoint to capture feedback. Test with the following:
`curl -X POST -u <user>:<pass> -d '{"email":"tyler@bocoup.com","response":"hiya"}' --header "Content-Type: application/json" https://ajl-staging.bocoup.org/api/feedback`

...also a backend endpoint to display the currently-logged-in user for pre-filling the frontend feedback form. Test with the following:
`curl -u <user>:<pass> https://ajl-staging.bocoup.org/session/me`

...also support for downloading feedback to a local CSV with the following:
`npm run download-feedback`

closes gh-152

### CODE REVIEW

**Comment by tkellen** [comment]
Note: the download-feedback script assumes you'll be running this for production. If you want to test out of the staging environment, you'll need to adjust the inventory file in package.json to be *staging*:
`download-feedback": "ansible-playbook --ask-vault-pass -i ansible/inventory/[[staging]] ansible/download-feedback.yml`

**Comment by tkellen** [comment]
Login URLs for invited users look like this:
https://tyler@bocoup.com:<password>@ajl-staging.bocoup.org/

Once logged in, you can confirm the API knows who you are here:
https://ajl-staging.bocoup.org/api/session/me

File: backend/endpoints/session/routes.js [hunk 1]
```diff
@@ -1,2 +1,10 @@
 export const location = '/api/session';
-export const routes = {};
+export const routes = {
+  get: {
```
Comment (gnarf) [inline backend/endpoints/session/routes.js]: I don't think we need this route at all right now, but there is no reason not to include it right now IMO.

Can you add a comment saying this is for the beta staging release if we want to remove it later.

Comment (tkellen) [inline backend/endpoints/session/routes.js]: Done!

**Pull request merged**